roles/installer/tasks/main.yml

```diff
@@ -1,10 +1,13 @@
 ---
-- name: Check for presence of Deployment
+- name: Check for presence of web and task deployments
   k8s_info:
     api_version: v1
     kind: Deployment
-    name: "{{ ansible_operator_meta.name }}"
+    name: "{{ item }}"
     namespace: "{{ ansible_operator_meta.namespace }}"
+  loop: 
+     - "{{ ansible_operator_meta.namespace }}-web"
+     - "{{ ansible_operator_meta.namespace }}-task"
   register: tower_deployment
```

Comment:
this doesnt work here u will just be overriding "tower_deployment"

Reply:
try to see what "tower_deployment" is being used for now and how it maps to "web" and "task" deployments

Reply:
Thank you! I was concerned about that. I will go investigate "tower_deployment."